src/engine/renderer/tr_local.h

```diff
@@ -1634,6 +1634,7 @@ enum class deluxeMode_t { NONE, GRID, MAP };
 
 		float          fovX, fovY;
 		matrix_t       projectionMatrix;
+		matrix_t       projectionMatrixNonPortal; // For skybox rendering in portals
```

Comment:
Why do skyboxes need to work differently here?

Reply:
Because portals set the near plane to the portal surfaces plane and far plane based on that, which results in the skybox getting clipped.